app/models/miq_vim_broker_worker/runner.rb

```diff
@@ -149,44 +149,58 @@ def drain_event
     event = @queue.deq
     return if event.nil?
 
+    ems_id = ems_ids_for_notify(event[:server], event[:username])
```

Comment:
Looks like the method gets its name from the `@ems_ids_for_notify` id cache here https://github.com/ManageIQ/manageiq/blob/570d2a41b16d3edb7f4084ddc15075ce4aa00ec0/app/models/miq_vim_broker_worker/runner.rb#L87
`get_ems_id_from_notify_event` or something similar would probably be a better name but this is how it is done currently.